README.md

```diff
@@ -43,3 +43,7 @@ A realistic example, using a public API, fetching data from NOAA
 
 ## configuration
 Shows how to use secrets
+
+## aws dynamo db authentication
```

Comment:
Hey @fivetran-chiragramachandraiah 
Can you add doc changes for key-based-replication and redshift examples here? I missed adding them